src/sentry/processing/backpressure/topology.py

```diff
@@ -0,0 +1,60 @@
+"""
+This defines the "topology" of our services.
+
+In other words, which service (consumer) depends on which other services (queues, processing store).
+"""
+
+from dataclasses import dataclass
+from typing import Mapping, Sequence, Union
+
+PROFILING_QUEUES = ["profiles.process"]
+PROCESSING_QUEUES = [
+    "events.preprocess_event",
+    "events.process_event",
+    "events.save_event",
+    "events.save_event_transaction",
+    "events.save_event_attachments",
+]
+SYMBOLICATION_QUEUES = [
+    "events.symbolicate_event",
+    "events.symbolicate_js_event",
+    "events.symbolicate_event_low_priority",
+    "events.symbolicate_js_event_low_priority",
+]
+REPROCESSING_QUEUES = [
+    "events.reprocess_events",
+    "events.reprocessing.preprocess_event",
+    "events.reprocessing.process_event",
+    "events.reprocessing.symbolicate_event",
+    "events.reprocessing.symbolicate_event_low_priority",
+]
+
+INGEST_QUEUES = PROCESSING_QUEUES + SYMBOLICATION_QUEUES
+
+ALL_QUEUES = PROFILING_QUEUES + PROCESSING_QUEUES + SYMBOLICATION_QUEUES + REPROCESSING_QUEUES
+
+# TODO: can someone help me fill this in?
+REDIS_STORES = ["default"]
```

Comment:
I think we should make this a key in `settings`, import it here, and override this in getsentry. This varies by configuration, and unfortunately there are some overrides in getsentry that make this hard to fetch from here. 

I'll leave pointers for the values here. Some of them may point to the same cluster.

- **Attachments**: 
    - [`rc-short` in Sentry](https://github.com/getsentry/sentry/blob/5cb760d8a7d4019adeceee9b5ba29de78aac1b7a/src/sentry/attachments/redis.py#L16)
    - [`processingstore-attachments` in getsentry](https://github.com/getsentry/getsentry/blob/c419be7ee063cd68028322c2707ec17987a2981a/getsentry/conf/settings/prod.py#L543-L544)
    - [`attachments` in single tenant](https://github.com/getsentry/getsentry/blob/c419be7ee063cd68028322c2707ec17987a2981a/getsentry/conf/settings/singletenant.py#L689-L692).
- **Event Processing Store**: 
    - Defaults to memcache [in Sentry](https://github.com/getsentry/sentry/blob/5cb760d8a7d4019adeceee9b5ba29de78aac1b7a/src/sentry/conf/server.py#L1798-L1799) (which we can ignore)
    - [`rc-processing` in getsentry](https://github.com/getsentry/getsentry/blob/c419be7ee063cd68028322c2707ec17987a2981a/getsentry/conf/settings/prod.py#L552)
    - [`processing` in single-tenant](https://github.com/getsentry/getsentry/blob/c419be7ee063cd68028322c2707ec17987a2981a/getsentry/conf/settings/singletenant.py#L249).
- **Locks**: First off, I'm not sure we should include this right away. We do use them in save_event though, so might not do us harm. 
    - [`default` in sentry](https://github.com/getsentry/sentry/blob/5cb760d8a7d4019adeceee9b5ba29de78aac1b7a/src/sentry/conf/server.py#L3305-L3308)
    - [`rc-default-cache` in getsentry](https://github.com/getsentry/getsentry/blob/c419be7ee063cd68028322c2707ec17987a2981a/getsentry/conf/settings/prod.py#L944-L947)
    - Can't find an override in single tenant, so probably `default`
- **Post Process Locks**:
    - [`default` in sentry](https://github.com/getsentry/sentry/blob/5cb760d8a7d4019adeceee9b5ba29de78aac1b7a/src/sentry/conf/server.py#L3310-L3313)
    - [`post-process-locks` in getsentry](https://github.com/getsentry/getsentry/blob/c419be7ee063cd68028322c2707ec17987a2981a/getsentry/conf/settings/prod.py#L911-L914)
    - no override in single tenant